www/sub/tablettab.html

```diff
@@ -32,8 +32,10 @@
           <div class="maslow-grid-item maslow-grid-item-btn" style="background-color: #f2f0e4;; color: #337ab7"
             onclick="sendMove('Z-');" id="zDown">
             <svg width="32" height="32" viewBox="0 0 38 38">
-              <polygon fill="currentColor" points="27.681 32.475 35.195 24.961 20.167 24.961 27.681 32.475" />
-              <rect fill="currentColor" height="2.5" transform="translate(11.262 44.101) rotate(-90)" width="21.79" x="16.786" y="15.17" />
+              <polygon transform="rotate(180) translate(-21 -44)" fill="currentColor"
```

Comment:
At first I was like "How did you fix both arrows by only changing one?" but then I realized only one needed to move to get them aligned. Nice work!